spec/features/admin/orders_spec.rb

```diff
@@ -285,6 +285,11 @@ def new_order_with_distribution(distributor, order_cycle)
         end
       end
 
+      scenario "cannot split the order in different stock locations" do
+        # There's only 1 stock location in OFN, so the split functionality that comes with spree should be hidden
+        expect(page).to_not have_selector '.split-item'
+      end
```

Comment:
Just for next time, I think a view spec it's much better suited for this. We're just checking the views output and not emulating a user's behavior at all.

Reply:
ok, that's interesting. It's just my ignorance, I've seen a view spec!
Would that be under a new folder /spec/views, something like [this](http://ruby-journal.com/how-to-write-rails-view-test-with-rspec/)?

Reply:
Exactly. I read about view specs, maybe I wrote one once but neither OFN nor Fair Food use them. You could argue that views should be so trivial that they don't require specs. And maybe there is another unit that is better tested for this? That said, I don't object view specs, they may be useful in some cases and way cheaper than feature specs.

Reply:
yes, I agree because views should be simple simple, but this is a very good example for a view spec.

Reply:
Yep, I have only written few of them but when you need them they come in handy! The Rails testing book talks about it in https://github.com/thoughtbot/testing-rails/blob/master/book/types_of_tests/view_specs.md.

Reply:
👌